internal/cli/apps.go

```diff
@@ -173,6 +174,34 @@ var CmdAppLogs = &cobra.Command{
 	},
 }
 
+// CmdAppExec implements the command: epinio apps exec
+var CmdAppExec = &cobra.Command{
+	Use:   "exec NAME",
+	Short: "creates a shell to the application",
+	Args:  cobra.ExactArgs(1),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		cmd.SilenceUsage = true
+
+		client, err := usercmd.New()
+		if err != nil {
+			return errors.Wrap(err, "error initializing cli")
+		}
+
+		// TODO: "--instance" ? to define the app instance (the pod)
+		// follow, err := cmd.Flags().GetBool("follow")
+		// if err != nil {
+		// 	return errors.Wrap(err, "error reading option --follow")
+		// }
+
+		err = client.AppExec(cmd.Context(), args[0])
+		if err != nil {
+			return errors.Wrap(err, "error getting a shell to application")
```

Comment:
Since errors.Wrap return `nil` in case of a nil error we can avoid the check (see the above function).

Reply:
oh nice, fixing.